Specs/NumberWithUnit/French/CurrencyModel.json

```diff
@@ -209,11 +209,11 @@
 }
 ,
 {
-  "Input": "le parc et sa famille a économisé pendant quatre ans pour acheter un appartement minuscule ici, mais a constaté que plus près ils sont arrivés à sauver 40,000 $ dont ils ont à l'origine eu besoin, plus du prix est monté.",
+  "Input": "le parc et sa famille a économisé pendant quatre ans pour acheter un appartement minuscule ici, mais a constaté que plus près ils sont arrivés à sauver 40000 $ dont ils ont à l'origine eu besoin, plus du prix est monté.",
   "NotSupportedByDesign": "python",
   "Results": [
     {
-      "Text": "40,000 $",
+      "Text": "40000 $",
```

Comment:
Similar case as above.

Reply:
Ideally would be "40 000" or "40.000" - it should match the spanish configuration.